.github/workflows/tests_secondary.yml

```diff
@@ -107,6 +107,13 @@ jobs:
     - uses: actions/checkout@v4
     - run: npm install -g yarn@1
     - run: npm install -g pnpm@8
+    - name: Setup Ubuntu Binary Installation # TODO: Remove when https://github.com/electron/electron/issues/42510 is fixed
+      if: ${{ runner.os == 'Linux' }}
+      run: |
+        if grep -q "Ubuntu 24" /etc/os-release; then
+          sudo sysctl -w kernel.apparmor_restrict_unprivileged_userns=0
```

Comment:
Does this actually help? We already run electron with `--no-sandbox` which is recommended as alternative solution in the internet. In github codespaces changing this option didn't have any effect on the tests for me.

Reply:
Not that it's relevant here, but what error were you seeing in Docker? If it's not the same image as the CI machines there may be additional dependencies that need to be installed.